scripts/commands/checkLinks.ts

```diff
@@ -63,8 +63,15 @@ async function loadFilesAndLinks(
       path.extname(filePath) === ".ipynb"
         ? markdownFromNotebook(source)
         : source;
+    const anchors = markdownLinkExtractor(markdown).anchors;
 
-    fileList.push(new File(filePath, []));
+    // Get the anchors from HTML ids.
+    const id_anchors = markdown.match(/(?<=id=")(.*)(?=")/gm);
+    if (id_anchors != null) {
+      id_anchors.map((id) => anchors.push("#" + id));
```

Comment:
`map` is intended for returning a new list

```suggestion
      id_anchors.forEach((id) => anchors.push("#" + id));
```